src/receivers/AwsLambdaReceiver.ts

```diff
@@ -197,6 +197,17 @@ export default class AwsLambdaReceiver implements Receiver {
     };
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  private getRawBody(awsEvent: AwsEvent): string {
+    if (typeof awsEvent.body === 'undefined' || awsEvent.body == null) {
+      return '';
+    }
+    if (awsEvent.isBase64Encoded) {
```

Comment:
Here is the way that Bolt for Python does the same: https://github.com/slackapi/bolt-python/blob/v1.6.1/slack_bolt/adapter/aws_lambda/handler.py#L84-L85

LGTM